.codeclimate.yml

```diff
@@ -0,0 +1,4 @@
+version: "2" # required to adjust maintainability checks
+checks:
+  method-count:
+    enabled: false
```

Comment:
I think general rule of thumb is to pull .codeclimate.yml from the IdP (https://github.com/18F/identity-idp/blob/master/.codeclimate.yml) and then slim down.

Reply:
Done